armory/data/datasets.py

```diff
@@ -21,11 +21,29 @@
 from art.data_generators import DataGenerator
 from armory.data.utils import curl, download_file_from_s3
 from armory import paths
+from armory.data.resisc45 import resisc45_split  # noqa: F401
```

Comment:
Could we move this inside of the resisc45 function? Or will that break tfds import hooking?